homeassistant/components/aftership/sensor.py

```diff
@@ -52,6 +52,7 @@
     {
         vol.Required(CONF_API_KEY): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_REFRESH_TIME, default=DEFAULT_REFRESH_TIME): cv.positive_int,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Thanks for the comment. 

Such change will take some time, how about I change default interval from 5 to 15 (without yaml config change) now and do the migration to UI flow in the other one?

5 to 15 change will fix the API request limit error for the free users of AfterShip, since now the widget doesn't work.